src/ProfileCard.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// region styled-component
+const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-left: 10px;
+`;
+
+const StyledImg = styled.img`
+  height: 70px;
+  width: 70px;
+  border-radius: 30%;
+  margin: 10px;
```

Comment:
여기 `margin` 때문에 메세지를 2개 이상 보내면 프사가 살짝 아래로 밀리는것 같습니다. 
```suggestion
  margin: 10px;
  margin-top: 0;
```

위쪽 마진만 없애면 아마 밀리지 않을 것 같다는 생각이 드네요...!

Reply:
오 적용해보니까 훨씬 이쁘게 되네요 감사합니다